contrib/core-contract-tests/tests/pox-4/pox_Commands.ts

```diff
@@ -63,6 +64,35 @@ export function PoxCommands(
         r.margin,
       )
     ),
+    // DelegateStackStxCommand
+    fc.record({
+      operator: fc.constantFrom(...wallets.values()),
+      // We can pick a constant from operator wallet.wasDelegatedBy: chain and tuple?
```

Comment:
Not sure I fully understand what we mean by chain and tuple?

Reply:
I mean the fast-check `chain` method. Is it the best way to do this in case we go for it? 
https://fast-check.dev/docs/core-blocks/arbitraries/combiners/any/#chain